web-registry/.env.example

```diff
@@ -25,3 +25,5 @@ REACT_APP_TOUCAN_LINK=https://mumbai.polygonscan.com
 REACT_APP_BRIDGE_CREDIT_CLASS_ID=C03
 # This is a flag to specify that Sentry should be initialized and enabled
 REACT_APP_SENTRY_ENABLED=
+# Feature flag to acticate bridge feature
+REACT_APP_BRIDGE=
```

Comment:
we should probably set this to true in netlify.toml for all contextes except prod